src/utils/createDynamicAppAtScriptTag.ts

```diff
@@ -23,21 +24,30 @@ export const createDynamicAppAtScriptTag = async (
 ): Promise<void> => {
   const { elementID, lang } = config
 
+  const scriptEl = getCurrentScript()
+  const props = getAllScriptURLParameters(scriptEl)
+  const merchantId = props.merchantId
+  const theme = merchantId
+
   if (elementID.slice(0, 1) !== '#') {
     throw new Error(
       'elementID must be a valid id selector string. E.g: #my-component'
     )
   }
 
-  const mountTargetSelector = `${elementID.substring(1)}-${lang}`
+  if (theme && !Object.values(ThemeEnum).includes(theme as ThemeEnum))
+    console.warn('theme doesnt exist for this merchantId')
```

Comment:
Please turn this into a block statement as it's easier to read when the condition is so long. Also, you should change this to `throw new Error` - look at the other errors we throw for inspiration on what would provide other developers with a clear error message

Reply:
removed this. Back to silently defaulting.